app/repositories/submission_repository.rb

```diff
@@ -1,18 +1,26 @@
 class SubmissionRepository
   def rejected
-    Submission.where(rejected: true)
+    Submission.where(rejected: true).order('created_at DESC')
   end
 
   def valid
     Submission.where(rejected: false)
```

Comment:
And if we do order it, this method shouldn't be reused because submissions would be ordered twice e.g. for `#rated_scope`